src/Halogen/HTML/Events.purs

```diff
@@ -121,11 +125,14 @@ onFileUpload
   => (t File -> i)
   -> IProp (onChange :: Event | r) i
 onFileUpload f = handler ET.change $
-  ( Event.target >=>
-    HTMLInputElement.fromEventTarget >=>
-    HTMLInputElement.files >>> unsafePerformEffect ) >>>
-  maybe none items >>>
-  f
+  ( Event.target
+      >=> HTMLInputElement.fromEventTarget
+      >=>
+        HTMLInputElement.files >>> unsafePerformEffect
```

Comment:
We can pull this up a line using parentheses, ie. `(HTMLInputElement.files >>> unsafePerformEffect)`